src/index.ts

```diff
@@ -4,30 +4,58 @@ import { Schemas } from "./schemas";
+import bgn_pcgn_alt from "./generated/bgn_pcgn_alt";
+import bs_2979 from "./generated/bs_2979";
+import bs_2979_alt from "./generated/bs_2979_alt";
+import gost_16876 from "./generated/gost_16876";
+import gost_16876_alt from "./generated/gost_16876_alt";
+import gost_52290 from "./generated/gost_52290";
+import gost_52535 from "./generated/gost_52535";
+import gost_7034 from "./generated/gost_7034";
+import gost_779 from "./generated/gost_779";
+import gost_779_alt from "./generated/gost_779_alt";
+import icao_doc_9303 from "./generated/icao_doc_9303";
+import iso_9_1954 from "./generated/iso_9_1954";
+import iso_9_1968 from "./generated/iso_9_1968";
+import iso_9_1968_alt from "./generated/iso_9_1968_alt";
+import mosmetro from "./generated/mosmetro";
+import mvd_310 from "./generated/mvd_310";
+import mvd_310_fr from "./generated/mvd_310_fr";
+import mvd_782 from "./generated/mvd_782";
+import scientific from "./generated/scientific";
+import telegram from "./generated/telegram";
+import ungegn_1987 from "./generated/ungegn_1987";
+import wikipedia from "./generated/wikipedia";
+import yandex_maps from "./generated/yandex_maps";
+import yandex_money from "./generated/yandex_money";
+
+export const ALA_LC = /*#__PURE__*/ Schema.load(ala_lc);
+export const ALA_LC_ALT = /*#__PURE__*/ Schema.load(ala_lc_alt);
+export const BGN_PCGN = /*#__PURE__*/ Schema.load(bgn_pcgn);
+export const BGN_PCGN_ALT = /*#__PURE__*/ Schema.load(bgn_pcgn_alt);
+export const BS_2979 = /*#__PURE__*/ Schema.load(bs_2979);
+export const BS_2979_ALT = /*#__PURE__*/ Schema.load(bs_2979_alt);
+export const GOST_16876 = /*#__PURE__*/ Schema.load(gost_16876);
+export const GOST_16876_ALT = /*#__PURE__*/ Schema.load(gost_16876_alt);
+export const GOST_52290 = /*#__PURE__*/ Schema.load(gost_52290);
+export const GOST_52535 = /*#__PURE__*/ Schema.load(gost_52535);
+export const GOST_7034 = /*#__PURE__*/ Schema.load(gost_7034);
+export const GOST_779 = /*#__PURE__*/ Schema.load(gost_779);
+export const GOST_779_ALT = /*#__PURE__*/ Schema.load(gost_779_alt);
+export const ICAO_DOC_9303 = /*#__PURE__*/ Schema.load(icao_doc_9303);
+export const ISO_9_1954 = /*#__PURE__*/ Schema.load(iso_9_1954);
+export const ISO_9_1968 = /*#__PURE__*/ Schema.load(iso_9_1968);
+export const ISO_9_1968_ALT = /*#__PURE__*/ Schema.load(iso_9_1968_alt);
+export const MOSMETRO = /*#__PURE__*/ Schema.load(mosmetro);
+export const MVD_310 = /*#__PURE__*/ Schema.load(mvd_310);
+export const MVD_310_FR = /*#__PURE__*/ Schema.load(mvd_310_fr);
+export const MVD_782 = /*#__PURE__*/ Schema.load(mvd_782);
+export const SCIENTIFIC = /*#__PURE__*/ Schema.load(scientific);
+export const TELEGRAM = /*#__PURE__*/ Schema.load(telegram);
+export const UNGEGN_1987 = /*#__PURE__*/ Schema.load(ungegn_1987);
+export const WIKIPEDIA = /*#__PURE__*/ Schema.load(wikipedia);
+export const YANDEX_MAPS = /*#__PURE__*/ Schema.load(yandex_maps);
+export const YANDEX_MONEY = /*#__PURE__*/ Schema.load(yandex_money);
```

Comment:
`#__PURE__` annotation informs Terser that an expression does not produce side effects. It allows to drop the code if it is not used in the final application bundle.